share/completions/imv.fish

```diff
@@ -0,0 +1,13 @@
+complete -c imv -s h --description 'Show help message and quit.'
```

Comment:
Please remove these trailing periods. They're annoying and take up a column.

Reply:
Done